packages/flutter/lib/src/material/app.dart

```diff
@@ -171,6 +173,31 @@ class MaterialApp extends StatefulWidget {
 
   /// {@macro flutter.widgets.widgetsApp.localizationsDelegates}
   ///
+  /// Internationalized apps that require translations for one of the locales
+  /// listed in [GlobalMaterialLocalizations] should specify this paramter
+  /// and list the [supportedLocales] that the application can handle.
+  ///
+  /// ```dart
+  /// import 'package:flutter_localizations/flutter_localizations.dart';
+  /// MaterialApp(
+  ///  localizationsDelegates: [
+  ///    // ... app-specific localization delegate[s] here
+  ///    GlobalMaterialLocalizations.delegate,
+  ///    GlobalWidgetsLocalizations.delegate,
+  ///  ],
+  ///  supportedLocales: [
+  ///     const Locale('en', 'US'), // English
+  ///     const Locale('he', 'IL'), // Hebrew
+  ///     // ... other locales the app supports
+  ///   ],
+  ///   // ...
+  /// )
+  /// ```
+  ///
+  /// The information that follows applies to the unusual case of an app
```

Comment:
Sounds like this should be under a new heading marked with `##` in the docs?

Reply:
Good idea. I'm tempted to call the section: "You Probably Don't Need to Read This" :-)